wordpress-mobile/gutenberg-mobile
# Improve accessibility on missing block

Fixes #950 

## To test

With VoiceOver/Talkback activated

1. Create a post with some unsupported blocks (gallery, contact form)
2. Tap on the unsupported gallery block, it should read `Unsupported block: Gallery`
3. Tap on the unsupported contact form (any non-core block), it should read `Unsupported block`
